util/godfig/src/backend/mod.rs

```diff
@@ -53,4 +56,11 @@ pub trait BackendOperations {
         T: serde::de::DeserializeOwned + serde::Serialize + Send,
         F: FnOnce(Option<T>) -> Fut + Send,
         Fut: std::future::Future<Output = Result<Option<T>, GodfigBackendError>> + Send;
+
+    async fn try_transaction_with_result<K, T, R, F, Fut>(&self, key: K, callback: F) -> Result<R, GodfigBackendError>
+        where
+        K: Into<Vec<String>> + Send,
+        T: serde::de::DeserializeOwned + serde::Serialize + Send,
```

Comment:
All import paths should be in use statements at the top of the file, this adds a lot of clutter.